### ISSUE TITLE
[FR] Copy and paste image support

### ISSUE THREAD

**Issue by liudonghua123** [AppFlowy-IO/AppFlowy#1323]
### Description

It would be nice if AppFlowy support paste image from clipboard.

### Impact

normal user which need to embed/insert some image in the document.

### Additional Context

the pasted image needs to save to somewhere and generate the image link in the document.

**Comment by cr0ssmind** [comment]
Please implement this feature. It is so useful - other document writing tools like Notable handles it very well.

This feature is essential me to switch onto appflowy permanently.

**Comment by rileyhawk1417** [comment]
@cr0ssmind do you mean just copying straight from your file manager then into AppFlowy?

**Comment by cr0ssmind** [comment]
@rileyhawk1417 Yes, and/or with other screenshotting tools, like under Windows with simple PrintScreen or "Windows+Shift+S" screen slicing.

**Comment by annieappflowy** [comment]
We support copying and pasting images in the latest release ([v0.3.0](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.3.0)).
Please give it try, @liudonghua123 @cr0ssmind  and let us know your feedback. Thanks!

**Comment by liudonghua123** [comment]
wow, I tried this new feature, good job, thanks!

**Comment by RobotNetwork** [comment]
I'm taking screenshots with ShareX and also can't paste them.

computerName: redacted
numberOfCores: 8
systemMemoryInMegabytes: 16384
userName: dahutchens
majorVersion: 10
minorVersion: 0
buildNumber: 22631
platformId: 2
csdVersion: 
servicePackMajor: 0
servicePackMinor: 0
suitMask: 256
productType: 1
reserved: 0
buildLab: 22621.ni_release.220506-1250
buildLabEx: 22621.1.amd64fre.ni_release.220506-1250
digitalProductId: [4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 203, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 19, 229, 203, 2, 0, 0, 49, 133, 210, 9, 0, 0, 0, 0, 21, 150, 32, 229, 203, 2, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 49, 133, 210, 9, 252, 0, 0, 0, 80, 165, 63, 229, 203, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 112, 180, 23, 229, 203, 2, 0, 0, 11, 0, 0, 0, 68, 1, 180, 0, 0, 0, 4, 0, 0, 0, 0, 0, 176, 181, 23, 229, 203, 2, 0, 0, 0, 0, 0, 0, 66, 5, 191, 0, 0, 0, 0, 0, 0, 0, 0, 0, 144, 178, 23, 229, 203, 2, 0, 0, 0, 0, 0, 0, 68, 1, 156, 0, 0, 0, 4, 0]
displayVersion: 23H2
editionId: Enterprise
installDate: 2024-08-28 21:39:33.000
productId: 00329-00000-00003-AA981
productName: Windows 11 Enterprise
registeredOwner: redacted
releaseId: 2009
deviceId: {36eec208-496e-4899-8cb0-71f9ead2bcf6}
Document: C:\Users\redacted\AppData\Roaming\io.appflowy\AppFlowy\data

**Comment by annieappflowy** [comment]
pasting with Win + V (Windows Clipboard History) doesn't work 
This is due to a Flutter bug we have no control of

**Comment by rileyhawk1417** [comment]
Not sure if this is still an issue with 0.8.1 on a deployed instance of AppFlowy in docker.
As for windows there is an issue on the Flutter repo about the [windows clipboard history](https://github.com/flutter/flutter/issues/143997)

**Comment by LucasXu0** [comment]
Pasting an image using the shortcut Ctrl/Cmd + V is now supported in the latest version.
